Sources/TypeDecoder/OrderedDictionary.swift

```diff
@@ -51,6 +51,17 @@ public struct OrderedDictionary<K: Hashable, V> {
         }
     }
 
+    /// subscript: get the value at the numeric index specified.
+    public subscript(index: Int) -> V? {
+        get {
```

Comment:
I don't think you need the `get{}` block.

Reply:
done